libraries/chem-meta/src/parsing-utils/molfile-handler.ts

```diff
@@ -101,7 +110,25 @@ export class MolfileHandler extends ChemicalTableParserBase implements ChemicalT
     return this.getNextColumnIdx(lineStartIdx);
   }
 
+  private isQuote(idx: number): boolean {
+    return this.fileContent.at(idx) === '\"' || this.fileContent.at(idx) === '\'';
+  }
+
+  private getNextIdenticalChar(idx: number): number {
+    const sym = this.fileContent.at(idx);
+    if (sym)
+      return this.fileContent.indexOf(sym, idx + 1);
+    else
+      return -1;
```

Comment:
you can rewrite it as a ternary, it would be more brief and clear:
`sym ? return this.fileContent.indexOf(sym, idx + 1) : return -1;`